### PR TITLE
Update data for sentiment analysis

### PR DESCRIPTION
**Pull request by jwood803**
Updates for #190.

@CESARDELATORRE I put the entire file in place of the three smaller ones. Is this ok or would it be best to download it when the sample runs if the file isn't on the local drive?

### CODE REVIEW

**Comment by justinormont** [comment]
I would recommend pulling this in realtime from the CDN. 

If you add it to the repo, you'll be increasing the git size by 75MB, and [won't be able to easily](https://stackoverflow.com/questions/2100907/how-to-remove-delete-a-large-file-from-commit-history-in-git-repository) remove it later. 

Another options for this is Git LFS.

File: samples/csharp/getting-started/BinaryClassification_SentimentAnalysis/SentimentAnalysis/SentimentAnalysisConsoleApp/Program.cs [hunk 1]
```diff
@@ -103,6 +105,25 @@ private static void TestSinglePrediction(MLContext mlContext)
             Console.WriteLine($"==================================================");
         }
 
+        private static void DownloadDataIfNotExists(string dataPath)
+        {
+            var downloadPath = "http://aka.ms/tlc-resources/benchmarks/WikiDetoxAnnotated160kRows.tsv";
```
Comment (justinormont) [inline samples/csharp/getting-started/BinaryClassification_SentimentAnalysis/SentimentAnalysis/SentimentAnalysisConsoleApp/Program.cs]: Likely my fault at somepoint in the past for giving a HTTP link. 
```suggestion
            var downloadPath = "https://aka.ms/tlc-resources/benchmarks/WikiDetoxAnnotated160kRows.tsv";
```

File: samples/csharp/getting-started/BinaryClassification_SentimentAnalysis/SentimentAnalysis/Data/wikipedia-detox-250-line-data.tsv [hunk 2]
```diff
@@ -1,250 +0,0 @@
-Sentiment	SentimentText
```
Comment (justinormont) [inline samples/csharp/getting-started/BinaryClassification_SentimentAnalysis/SentimentAnalysis/Data/wikipedia-detox-250-line-data.tsv]: I see other references to this dataset:
https://github.com/dotnet/machinelearning-samples/search?q=wikipedia-detox-250-line-data.tsv&unscoped_q=wikipedia-detox-250-line-data.tsv

The reference may have their own copy.

**Comment by CESARDELATORRE** [comment]
Ankit wanted for this sample to be able to run it off-line, for anyone trying it.
So, we're going to cut the larger dataset to around 40k records. We've been testing it with that training and it trains pretty quick (acceptable for a getting started, around 10 or 15 secs or so) and any user will be able to run it offline. Also 40k records is a significant smaller file.

**Comment by justinormont** [comment]
If you're looking to cut down the size of the dataset, I'd recommend ~10k rows. 

Num rows vs. metrics:  ([learning curves](https://en.wikipedia.org/wiki/Learning_curve_(machine_learning)))
```md
Rows  Time  Trainer                    Accuracy      AUC    AUPRC  F1-score
160k   34s  AveragedPerceptronBinary     0.9597   0.9645   0.8538    0.7780
 40k   11s  AveragedPerceptronBinary     0.9528   0.9579   0.8139    0.7541 
 10k    6s  AveragedPerceptronBinary     0.9545   0.9471   0.8188    0.7729
  5k    5s  AveragedPerceptronBinary     0.9492   0.9384   0.8094    0.7652
```
My pipline 
10k rows is 4.2MB. Cutting to 1k rows means you'll need CV mode to get accurate scores (as the 80-20 split would leave 200 rows for scoring). 

For this dataset, I would like to see an AUC of ~97%.

**Comment by prathyusha12345** [comment]
I will merge the changes.

But I am going to replace with **40k** rows of data in data set file and remove the DownloadDataIfNotExists() method. I will place the data set file in **Data** folder.

**Pull request merged**